Mathlib/Algebra/LinearRecurrence.lean

```diff
@@ -147,13 +147,9 @@ theorem sol_eq_of_eq_init (u v : ℕ → α) (hu : E.IsSolution u) (hv : E.IsSol
   ext x
   exact mod_cast h (mem_range.mpr x.2)
 
-/-! `E.tupleSucc` maps `![s₀, s₁, ..., sₙ]` to `![s₁, ..., sₙ, ∑ (E.coeffs i) * sᵢ]`,
-  where `n := E.order`. This operation is quite useful for determining closed-form
-  solutions of `E`. -/
-
-
 /-- `E.tupleSucc` maps `![s₀, s₁, ..., sₙ]` to `![s₁, ..., sₙ, ∑ (E.coeffs i) * sᵢ]`,
-  where `n := E.order`. -/
+where `n := E.order`. This operation is quite useful for determining closed-form
+solutions of `E`. -/
```

Comment:
There was a file-level docstring (shown in the docs) and a def docstring. I'm not sure a linter only PR is the place to change that.

Reply:
Sure, I've reverted anything but the indentation changes.